js/templates.js

```diff
@@ -0,0 +1,18 @@
+import { creatingDuplicatesPictures } from './data.js';
```

Comment:
Тут следует поменять на 
```suggestion
import { createDuplicatesPictures } from './data.js';
```